database_schema.md

```diff
@@ -75,6 +75,29 @@ person_id: {
 }
 ```
 
+### Body
+```
+body_id: {
+    name: str
+    description: str
+    created: datetime
+    ended: datetime
```

Comment:
What would be `created` and `ended` datetime values for City Council and Council Briefing bodies?
And how would we query to get all the active bodies?

Reply:
I was thinking that any body that is permanently ongoing can have be null/empty for those fields. And we could query for whether `ended` is present if we want the active bodies.

If storing null/empty values isn't ideal or best practice we can switch to `is_active: bool` instead.

Reply:
Querying for null is kind of icky in Firebase. Plus we would have to query with the condition: `ended == null or today's date < ended`, to get all the active bodies.

With `is_active`, we need only one condition in query

Reply:
I like `is_active` a bit more than just having `ended`. Having all three, while duplicated data, is "simpler" to understand.

Reply:
I'm OK with having all three fields.